i-dot-ai/evaluation-registry
# Feature/update crown logo

## Summary

This work has been done to update the Crown logo in the the header and favicon. Jira ticket ER-131.

As part of this, the gov.uk frontend has been upgraded from v4.7.0 to v4.8.0. Other icons (e.g. Apple touch icons) have also been updated.

## How to test

Navigate to any page. Check that the new crown logo appears in the header and favicon. The new logo is rounded at the top while the previous version had a slight dip.

### Before:
![image](https://github.com/i-dot-ai/evaluation-registry/assets/1634605/2300e093-6a96-44f0-8fcb-eaedf614515c)

### After:
![image](https://github.com/i-dot-ai/evaluation-registry/assets/1634605/05490898-4ceb-4eca-b3cf-928c13ecce75)
